python/pyproject.toml

```diff
@@ -32,9 +32,8 @@ dynamic = ["version"]
 requires-python = ">=3.10"
 description = "Python library for Apache Arrow"
 readme = {file = "README.md", content-type = "text/markdown"}
-license = {text = "Apache Software License"}
+license = "Apache-2.0"
 classifiers  = [
-    'License :: OSI Approved :: Apache Software License',
```

Comment:
Is there a reason to remove the classifier?

Reply:
Ok, it seems that NumPy have [removed their license classifier](https://github.com/numpy/numpy/commit/0f39fd27ff149b25f74fc58d589c92b4fd881226), so I'm fine with that as well.